content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -248,6 +248,26 @@ client.query('SELECT $1::text as message', ['Hello world!'], (err, result) => {
 
 {{< /tabs >}}
 
+## Explore the APM Connection
+
+### Attribute Active Database Connections to the Calling APM Services
+
+{{< img src="database_monitoring/dbm_apm_active_connections_breakdown.png" alt="View active connections to a database broken down by the APM Service they originate from.">}}
+
+Break down active connections for a given host by the upstream APM services making the requests. You can attribute load on a database to individual services to understand which services are most active on the database. Pivot to the most active upstream service’s service page to continue the investigation.
+
+### View Query Samples from the Traces of the Calling Requests
+
+{{< img src="database_monitoring/dbm_query_sample_trace_preview.png" alt="Preview the sampled APM trace that the query sample being inspected was generated from.">}}
+
+When viewing a Query Sample in Database Monitoring, if the associated trace has been sampled by APM, you can view the DBM Sample in the context of the APM Trace. This allows you to combine DBM telemetry, including the explain plan and historical performance of the query, alongside the lineage of the span within your infrastructure to understand if a change on the database is responsible for poor application performance.  
```

Comment:
I think the screenshot that goes along with this is the trace in the context of DBM, not the other way around? Should this be a screenshot of the Query Details tab in APM instead?